### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by penguin1109** [facebookresearch/mae#164]
How many epochs do we need to train in order to make the model generate reasonable ouputs? 
I am currently training the MAE model from scratch, with 4000 images (the dataset is very small though). But the model keeps on generating images like the image uploaded.

What is the possible problem here??

Thanks.
![0 (1)](https://user-images.githubusercontent.com/59923223/235821349-36f37a44-1a76-4a30-8d87-5f4e73c44eb8.png)

**Comment by johnWang2022** [comment]
I do not know you set the norm pix loss is true or false , when you make pretraining.  If you want to generate the image like original image  ,  you should set norm pix loss to false ( remove the --norm_pix_loss) when you make pretraining or fine-tuning. 
it will close the norm for patch of target tensor.
![1684404187983](https://github.com/facebookresearch/mae/assets/105700302/d70ca840-9ce2-4d9b-86cf-4e535c40ff0f)
